src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js

```diff
@@ -8,6 +8,11 @@ modules.define('middleware__html-cache', ['logger', 'constants'], function (prov
     provide(function () {
         return function (req, res, next) {
             var pagePath = path.join(constants.PAGE_CACHE, req.__data.node.url, req.lang + '.html.gzip');
+            
+            if (req.disableCache) {
+                next();
```

Comment:
Yes, you are right. I fix it ASAP in next commit https://github.com/bem/bem-site-engine/commit/3eb8a8be116edcc95bd8e24addb71220a28ddf41